lib/routes/mashiro/index.ts

```diff
@@ -0,0 +1,61 @@
+import { Route } from '@/types';
+import { namespace } from './namespace';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+const baseUrl = `https://${namespace.url}`;
+
+export const route: Route = {
+    path: '/:lang',
+    categories: ['blog'],
+    example: '/mashiro/en',
+    parameters: { lang: 'the language of the site. Can be either `en` or `zh-cn`. Default: `en`' },
+    radar: [
+        {
+            source: ['mashiro.best/', 'mashiro.best/:lang/'],
+        },
+    ],
+    name: `Mashiro's Baumkuchen`,
```

Comment:
Route name should not be the same as namespace name https://github.com/DIYgod/RSSHub/blob/6c2a2389b6e750b3d73de068c08b10936f1c492a/lib/routes/mashiro/namespace.ts#L4

Reply:
OK, I changed it to `Blog`.